README.md

```diff
@@ -87,6 +87,64 @@ Start the containers using docker compose:
 docker-compose up
 ```
 
+### Testing
+
+#### Running the standard test-suite
+
+running locally, in the `osmaxx-py` directory (using a virtualenv):
+
+```shell
+DJANGO_SETTINGS_MODULE=config.settings.test ./manage.py test
+```
+
+*Notice*: Since the ampq Service is not available, you'll get a 4 failing tests. Therefore it is recommended to
+run them in a docker container, as described below.
+
+Using docker and docker-compose:
+ 
+```shell
+docker-compose run --rm webappdev bash -c 'DJANGO_SETTINGS_MODULE=config.settings.test ./manage.py test'
+```
+
+#### Running the test-suite and integration tests
+
+There's a script to facilitate testing including some docker integration tests:
+
+```shell
+./test.sh
+```
+
+#### Running end to end tests (e2e)
+
+**Warnings**: 
```

Comment:
Warnings should be a subsection is what you mean, right? We could put the entire documentation on testing into a separate file, and have a link to there, if that is what would make more sense.